src/components/grid/grid.xml

```diff
@@ -56,7 +56,7 @@
         onMouseWheel.bind="onMouseWheel"
         onClosePopover.bind="onClosePopover"
       />
-      <t t-if="env.model.getters.getEditionMode() === 'inactive' and isAutoFillActive()">
```

Comment:
`Autofill` component needs to be always active for tooltips and handler to work properly when edge scrolling

Reply:
> when it's outside the viewport, it's actually still visible at the very top-left grid corner ![aa](https://user-images.githubusercontent.com/10908305/215501194-50f2e329-6724-4265-b2e8-f55599936ab2.png)

@LucasLefevre Good eyes 😄 I'll change it to `visibility:hidden`.